pdc/apps/auth/management/commands/updateresources.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+import inspect
+
+from django.core.management.base import BaseCommand
+from django.utils.module_loading import autodiscover_modules
+
+from pdc.apps.auth.models import ResourcePermission, ActionPermission, Resource
+from pdc.apps.utils.SortedRouter import router
+from pdc.apps.utils.utils import convert_method_to_action
+
+
+class Command(BaseCommand):
+    help = "Updates list of resources for which permissions can be created"
+    API_WITH_NO_PERMISSION_CONTROL = set(['auth/token', 'auth/current-user'])
+
+    def handle(self, *args, **options):
```

Comment:
Is it possible to call this automatically on `pre_migrate` signal rather than creating new command?

Reply:
Good suggestion. It might be possible. I'll need to make sure the signal is fired even if there are no actual migrations to run.

Reply:
It works, the signal is fired even if there is no change. I used `post_migrate` to make sure that the database schema matches the models. Would `pre_` be better?

Reply:
I guess `post_` is OK, not sure if any migrations would want to access it.